### DISCUSSION TITLE
reconfigure command MQTT

### DISCUSSION THREAD

**Discussion by bacobas** [Koenkk/zigbee2mqtt#17382] [General]
Actually we develop a #backend application that communicate in MQTT with Zigbee2Maqtt platform 
and we need to integrate the “#reconfigure” command ( as the Zigbee2Maqtt frontend application)
in our backend application : 
All that for let us able to send the same command by our application  tom the Zigbee End device .
How can we do that?
Please help us. 
BR

**Comment by sjorge** [comment]
https://www.zigbee2mqtt.io/guide/usage/mqtt_topics_and_messages.html#zigbee2mqtt-bridge-request
